swagger-spec/logs/detail.yaml

```diff
@@ -1 +1,67 @@
 # /logs/{log_id}/
+get:
+  summary: Retrieve a Log
+  description: >-
```

Comment:
Can you update the description of this endpoint to better match the style used [here](https://github.com/caseyrollins/swagger-ui/blob/master/swagger-spec/preprints/detail.yaml#L4)?

It would also be nice to have a general explanation of what logs are and how they're useful (permanent immutable record of a node's history) in the first part of the description.